app/publishing/delinkedCalculation/get-unpublished.js

```diff
@@ -0,0 +1,8 @@
+const getUnpublishedDelinked = require('./get-unpublished-delinked')
+
+const getUnpublished = async (transaction) => {
+  const delinked = await getUnpublishedDelinked(transaction)
+  return delinked
```

Comment:
As discussed, feels like this is adding a file for file's sake, but appreciate as you addressed there is value in keeping it this way.